Develop

Kanezoor opened the pull request:
https://kanezoor.github.io/layout_landing-page/

Kanezoor committed:
saved for transport

Kanezoor committed:
saved for transportation

Kanezoor committed:
Merge pull request #1 from Kanezoor/develop

Develop

Kanezoor committed:
prepared to moving

Kanezoor committed:
Merge pull request #2 from Kanezoor/develop

prepared to moving

Kanezoor committed:
final result

Kanezoor committed:
push to the origin

Kanezoor committed:
Merge pull request #3 from Kanezoor/develop

final result from BEM modul

Kanezoor committed:
added oferflow:hidden for menu

Kanezoor committed:
Merge pull request #4 from Kanezoor/develop

added oferflow:hidden for menu

Kanezoor committed:
WIP: save progress

Kanezoor committed:
close to finish

Kanezoor committed:
added couple of images

Kanezoor committed:
added final touches

2pasha reviewed (changes requested):
great job! 👏

please take a look on these improvements: 

1. these elements should be in svg format
<img width="549" height="104" alt="image" src="https://github.com/user-attachments/assets/746df6f2-36e1-46f9-a6cb-98f550550344" />

2. when user hover on phone icon - phone number should be appeared
<img width="243" height="114" alt="image" src="https://github.com/user-attachments/assets/f13feb99-454a-4e9c-954c-ac970502dfb9" />

3. please add favicon

4. after form submitting - user have received 450 error. Instead of error user should see imitation of form submitting (reset all fields and do not refresh the page) 
<img width="740" height="163" alt="image" src="https://github.com/user-attachments/assets/ed8b6add-b495-44e1-b00e-fd46d654015e" />

5. here should be link to google maps
<img width="535" height="298" alt="image" src="https://github.com/user-attachments/assets/854ef444-c734-4a1c-a1cd-9a03a412fa4f" />

Kanezoor committed:
fixed issues from code review

etojeDenys reviewed (changes requested):
almost done

1. phone number style looks different with the layout
<img width="188" height="96" alt="image" src="https://github.com/user-attachments/assets/16f360e8-59e7-4cb8-9fbe-49eeea10c82e" />

figma:
<img width="154" height="59" alt="image" src="https://github.com/user-attachments/assets/cddf078e-e484-494c-a239-26e9f49aeb7a" />

2. phone number should only show up on the desktop version

Kanezoor committed:
changed a phone number on phone icon

etojeDenys reviewed (approved):
good job

Kanezoor committed:
added a preview picture

Kanezoor committed:
Revise README for Nothing Phone landing page

Updated the README to reflect the project as a landing page for the Nothing Phone concept, added key features, and provided instructions for local setup.

Kanezoor committed:
Fix image filename and format demo link

Corrected image file name and updated demo link formatting.

Kanezoor committed:
Format semantic HTML section in README

Kanezoor committed:
Add creator information to README
